src/Tizen.NUI/src/internal/Interop/Interop.Renderer.cs

```diff
@@ -96,52 +96,52 @@ internal static partial class Renderer
             public static extern int Renderer_Property_STENCIL_OPERATION_ON_Z_PASS_get();
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_new_Renderer_Property")]
-            public static extern global::System.IntPtr new_Renderer_Property();
+            public static extern global::System.IntPtr NewRendererProperty();
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_delete_Renderer_Property")]
-            public static extern void delete_Renderer_Property(global::System.Runtime.InteropServices.HandleRef jarg1);
+            public static extern void DeleteRendererProperty(global::System.Runtime.InteropServices.HandleRef jarg1);
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Renderer_New")]
-            public static extern global::System.IntPtr Renderer_New(global::System.Runtime.InteropServices.HandleRef jarg1, global::System.Runtime.InteropServices.HandleRef jarg2);
+            public static extern global::System.IntPtr RendererNew(global::System.Runtime.InteropServices.HandleRef jarg1, global::System.Runtime.InteropServices.HandleRef jarg2);
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_new_Renderer__SWIG_0")]
-            public static extern global::System.IntPtr new_Renderer__SWIG_0();
+            public static extern global::System.IntPtr NewRendererSwig0();
```

Comment:
How about removing "SwigXXX" word? it has came from auto SWIG c# binding.

Reply:
If I remove SwigXXX, then how do I differentiate between NewRendererSwig0 and NewRendererSwig1?